src/VisualStudio/CSharp/Impl/ProjectSystemShim/CSharpProjectShim.ICSInputSet.cs

```diff
@@ -45,6 +45,8 @@ public void SetWin32Resource(string filename)
 
         public void SetOutputFileName(string filename)
         {
+            VisualStudioProjectOptionsProcessor.SetOutputFileName(filename);
+
             VisualStudioProject.IntermediateOutputFilePath = filename;
```

Comment:
It feels like either:

1. We should move the set of IntermediateOutputFilePath into SetOutputFileName; otherwise we're in a world where they can get out of sync.
2. Just delete IntermediateOutputFilePath, and just make the options processor responsible for it?

I guess is anything using IntermediateOutputFilePath at this point? Or can we simply delete it?

Reply:
I moved the code updating compiler outputs to `IntermediateOutputFilePath` property setter.